Makefile

```diff
@@ -87,17 +87,17 @@ docs/%.5: docs/%.5.md
 
 docs: $(MANPAGES_MD:%.md=%)
 
-install: ocid.conf
+install:
```

Comment:
Why drop the requirement? In fact, we should add `binaries` and `docs` requirements as well.